reference/clinic/models/tide/tideConfig.v1.yaml

```diff
@@ -23,6 +23,10 @@ properties:
     description: Threshold used for determining if a value is very high
     type: number
     x-go-type: float64
+  extremeHighGlucoseThreshold:
+    description: Threshold used for determining if a value is extremely high
```

Comment:
None of these tell me details... Are they inclusive, or exclusive? Does the number need to be higher than this, or lower than this? You didn't really add all these, but since you're here, you might consider improving these.